packages/overlay/src/telemetry/store/slices/sharedSlice.ts

```diff
@@ -1,8 +1,8 @@
 import type { StateCreator } from "zustand";
-import type { SentryStore, SharedSliceActions } from "~/integrations/sentry/store/types";
-import type { SentryErrorEvent } from "~/integrations/sentry/types";
-import { getNativeFetchImplementation } from "~/integrations/sentry/utils/fetch";
 import { log } from "~/lib/logger";
+import type { SentryStore, SharedSliceActions } from "~/telemetry/store/types";
+import type { SentryErrorEvent } from "~/telemetry/types";
+import { getNativeFetchImplementation } from "~/telemetry/utils/fetch";
```

Comment:
This and the "local event transport" system should also be yanked as they make no sense in a world where Spotlight is not embedded.